File: VB/Helpers/DataHelper.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Data
Imports System.Linq

Namespace dxQ529392
	Public Class DataHelper
		Public Shared Function GetDataSource() As Object
			Dim table = New DataTable()
			table.Columns.Add(New DataColumn("ID"))
			table.Columns.Add(New DataColumn("Name"))
			table.Columns.Add(New DataColumn("SecondName"))
			table.Columns.Add(New DataColumn("Adress"))
			table.Columns.Add(New DataColumn("PhoneNum"))
			table.Columns.Add(New DataColumn("Personal information"))
			Return table
		End Function
	End Class
End Namespace

File: CS/Helpers/DataHelper.cs
using System;
using System.Collections.Generic;
using System.Data;
using System.Linq;

namespace dxQ529392 {
    public class DataHelper {
        public static object GetDataSource() {
            var table = new DataTable();
            table.Columns.Add(new DataColumn("ID"));
            table.Columns.Add(new DataColumn("Name"));
            table.Columns.Add(new DataColumn("SecondName"));
            table.Columns.Add(new DataColumn("Adress"));
            table.Columns.Add(new DataColumn("PhoneNum"));
            table.Columns.Add(new DataColumn("Personal information"));
            return table;
        }
    }
}
